understanding/20/error-identification.html

```diff
@@ -25,12 +25,11 @@ <h2>Intent of Error Identification</h2>
          has occurred and can determine what is wrong. In the case of an unsuccessful form submission,
          it is not sufficient to only re-display the form without providing any hint that the submission
          failed.
-         The error must be indicated in <a>text</a>.
-         Whether or not an error message provides users with sufficient information about the nature of
-         the error, and what they should do to correct it, is covered more specifically by
-         <a href="error-suggestion">3.3.3 Error Suggestion</a>.
+         The error must be indicated in <a>text</a>.</p>
+         <p>This SC requires that users be provided with information about the nature of the error, including the identity of the item in error. What the user should do to correct the item in error is covered by 
+         <a href="error-suggestion">3.3.3 Error Suggestion</a>. Often, the error description can be phrased so that it meets both SC 3.3.1 and SC 3.3.3 at the same time. For instance, "Please provide a valid email address" conveys that the email field is empty or invalid, and also how it can be fixed.
       </p>
-      <p>An "input error" is information provided by the user that is not accepted. This includes:</p>
+      <p>An "input error" is input data that has failed validation. This includes:</p>
```

Comment:
```suggestion
      <p>An "input error" is caused by information provided by the user that has failed validation. This includes:</p>
```

Reply:
I think we can remove this sentence entirely, as the normative section already links to the definition: [input error](https://www.w3.org/WAI/WCAG22/Understanding/error-identification.html#dfn-input-error)

IMO, we can remove enirely:

> An "input error" is information provided by the user that is not accepted. This includes:
> - information that is required by the web page but omitted by the user, or
> - information that is provided by the user but that falls outside the required data format or allowed values.

And replace the sentence below:

> For example:

with:

> Examples of "input error" include:

Reply:
Giacomo's suggestion, slightly modified.
```suggestion
      <p>An "input error" includes:</p>
```